src/Rebing/GraphQL/Support/Field.php

```diff
@@ -210,10 +217,7 @@ public function getAttributes()
             'args' => $this->args(),
         ], $attributes);
 
-        $type = $this->type();
-        if (isset($type)) {
-            $attributes['type'] = $type;
-        }
+        $attributes['type'] = $this->type();
```

Comment:
This is the code change mentioned in the PR description; a side effect of adding the type and phpstan error reporting.